runtime/vm/BufferWriter.hpp

```diff
@@ -36,6 +36,7 @@ class VM_BufferWriter {
 	U_8 *_bufferEnd;
 	U_8 *_maxCursor;
 	bool _overflow;
+	J9PortLibrary *privatePortLibrary;
```

Comment:
This field should use the same naming convention as others (and should specifically avoid the special name `privatePortLibrary`); suggest it should be also first to match the constructor;
```c
	private:
	J9PortLibrary *_portLibrary;
	U_8 *_buffer;
```

Reply:
Fixed.